brigade/templates/base.html

```diff
@@ -21,15 +21,9 @@
   <body id="{% block page_id %}{% endblock %}">
 
     <div class="js-container">
-      <div class="global-header global-header--above">
-        <a href="https://www.codeforamerica.org/fellowship" target="_blank">
-          Want to improve your community? Apply for the
-          <span class="global-header__above-underline">
-            Code for America Community Fellowship
-          </span>
-          by April 30th.
-        </a>
-      </div>
+
+      <nav class="nav-global-primary">
+      </nav>
```

Comment:
I'm curious what the empty `nav.nav-global-primary` element is for -- is there a plan to use it later?

Reply:
No idea...